homeassistant/components/media_player/sonos.py

```diff
@@ -42,6 +42,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     players = None
     hosts = config.get('hosts', None)
+    group_devices = config.get('group_devices', False)
```

Comment:
I wonder if the key `group_devices` is descriptive of what is happening. What about `groups_only` ?

Or, and this is based on not knowing exactly how Sonos works: would it be possible to change functionality to only add group coordinators and any device without a group? Since group-members that are not the coordinator are not very useful to add in the first place anyway?